package.json

```diff
@@ -152,7 +152,7 @@
 	},
 	"dependencies": {
 		"lodash": "^4.17.21",
-		"pluralize": "^8.0.0",
+		"pluralize": "github:plurals/pluralize",
```

Comment:
Nice and simple! 
However it worries me a bit because it's not fixed against any version of the lib, it will always use the most recent one which will be resolved when I build the package for release. This might potentially break stuff in the future.
Luckily there is an easy fix for that: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#github-urls
You can use specific commit, so we can snapshot this dependency to the most recent commit as of today and then we're safe.

Reply:
I was using it in my library `const-name`, however then I remember that intallations happen by using provided lockfile, so I checked that lockfile has specific commit locked, but ok I will also lock it here.

Reply:
Well, you're actually right, I missed that lockfile change because it was collapsed by default. However I still didn't make the GitHub Actions deploy to work correctly and when I deploy locally I usually use `npm install` instead of `npm ci` so it will be safer to have it here as well.